packages/web/config/ibc-assets.ts

```diff
@@ -163,6 +163,44 @@ export const IBCAssetInfos: (IBCAsset & {
     coinMinimalDenom: "uscrt",
     isVerified: true,
   },
+  {
+    counterpartyChainId: "secret-4",
+    sourceChannelId: "channel-476",
```

Comment:
Won't treat this as a blocker, but we should have this channel registered to chain reg as well...

Reply:
Cc @DrPresident

Reply:
@JeremyParish69 This PR is ready https://github.com/cosmos/chain-registry/pull/997

Reply:
@mohammedpatla still seems to be some errors... Looks like the connection info submitted here and to Chain Registry isn't quite right...

Reply:
Should be fixed now on master